frontend/cypress/support.js

```diff
@@ -6,7 +6,7 @@ Cypress.Screenshot.defaults({
 });
 
 beforeEach(() => {
-  cy.visit('/');
+  cy.visit('/', { timeout: 120000 });
```

Comment:
Instead of increasing this, it is possible to to use `try-catch` to handle `cy.get` timeouts?

Reply:
yes, use a try-catch to bypass the errors.

Reply:
As far as I know, we can't bypass the errors from `cy.visit()` in this case. See [here](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#You-cannot-add-a-catch-error-handler-to-a-failed-command)